bigbluebutton-html5/imports/ui/components/dropdown/trigger/component.jsx

```diff
@@ -41,7 +41,17 @@ export default class DropdownTrigger extends Component {
   }
 
   render() {
-    const { children, className, ...restProps } = this.props;
+    const remainingProps = Object.assign({}, this.props);
```

Comment:
`const remainingProps = { ...this.props }`

Reply:
Hey, I was using [your own method](https://github.com/bigbluebutton/bigbluebutton/blame/v2.0.x-release/bigbluebutton-html5/imports/ui/components/button/component.jsx#L129) to remove the unknown props here.

Reply:
I was not complaining about the deletion of props.